index.html

```diff
@@ -63,8 +221,39 @@ <h1>I'm Alusine, glad to meet you!</h1>
               <a href="#"><img src="images/Vector.svg" alt="other" /></a>
             </li>
           </ul>
+          <div class="get-resume">
+            <a href="#">Get my resume</a>
+          </div>
         </div>
-      </div>
-    </section>
+        <section class="second-section bg-white">
+          <div class="lang-dropdown">
+            <h3>Languages</h3>
+            <span class="dropdown"></span>
+          </div>
+          <div class="technologies">
+            <div class="tech">
+              <img src="images/icjscript.png" alt="dropdown" />
+              <p>Javascript</p>
+            </div>
+            <div class="tech">
+              <img src="images/ichtml.png" alt="dropdown" />
+              <p>Html</p>
+            </div>
+            <div class="tech">
+              <img src="images/iccss.png" alt="dropdown" />
+              <p>CSS</p>
+            </div>
+          </div>
+          <div class="lang-dropdown border-bottom">
+            <h3>Frameworks</h3>
+            <span class="drop-right"></span>
+          </div>
+          <div class="lang-dropdown border-bottom">
+            <h3>Skills</h3>
```

Comment:
It would be better if you also consider the following suggestions: 
- [ ] The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`.
- [ ] The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`